src/hdmf/common/__init__.py

```diff
@@ -20,6 +20,30 @@
 # a global type map
 global __TYPE_MAP
 
+@docval({'name': 'config_path', 'type': str, 'doc': 'Path to the configuration file.'},
+        is_method=False)
+def load_type_config(**kwargs):
+    """
+    This method will either load the default config or the config provided by the path.
+    """
+    config_path = kwargs['config_path']
+    __TYPE_MAP.ts_config.load_type_config(config_path)
+
+def get_loaded_config():
```

Comment:
```suggestion
def get_loaded_type_config():
```
For consistency, I think we should either use "type config" or just "config". I think "type config" is fine.